L1Trigger/L1TMuonOverlapPhase1/interface/Omtf/OMTFinput.h

```diff
@@ -27,6 +29,7 @@ class OMTFinput : public MuonStubsInput {
   ///Method used in DiMuon studies.
   void mergeData(const OMTFinput *aInput);
 
+  //TODOO make it: if the layer is bending layer, the phiB from the iLayer -1 is returned, change in all places it is used
```

Comment:
This comment points to a possible improvement (clean up) to the codes, which would allow avoiding "if's" like e.g. in [ProcessorBase.cc#L156](https://github.com/cms-sw/cmssw/blob/c4e19155815230c34f92d11ab87d8f34db0cfbac/L1Trigger/L1TMuonOverlapPhase1/src/Omtf/ProcessorBase.cc#L156)
However it would have to be done carefully, so it is postponed... but the TODO reminds us about it.